aptos-move/framework/aptos-framework/sources/configs/gas_schedule.move

```diff
@@ -0,0 +1,55 @@
+/// This module defines structs and methods to initialize VM configurations,
+/// including different costs of running the VM.
+module aptos_framework::gas_schedule {
+    use std::error;
+    use std::string::String;
+
+    use aptos_framework::reconfiguration;
+    use aptos_framework::system_addresses;
+    use aptos_framework::timestamp;
+    use aptos_framework::util::from_bytes;
+
+    /// Error with config
+    const ECONFIG: u64 = 0;
+    /// The provided gas constants were inconsistent.
+    const EGAS_CONSTANT_INCONSISTENCY: u64 = 1;
+
+    struct GasEntry has store, copy, drop {
+        key: String,
+        val: u64,
+    }
+
+    struct GasSchedule has key, copy, drop {
+        entries: vector<GasEntry>
+    }
+
+    public fun initialize(account: &signer, gas_schedule_blob: vector<u8>) {
+        timestamp::assert_genesis();
```

Comment:
Do we need this check? I'd ideally prefer not to have it unless it's absolutely critical. This doesn't look like it can be called more than once anyway (the assert on line 30 will fail)

Reply:
> we might start setting timestamp early in the genesis flow, which will break all of these asserts

Well if that's the where we'll eventually want to be, then I'm fine removing this assertion.

Reply:
But maybe let's keep them for now so they are consistent with some other stuff like ChainId? You can remove them when you when you revisit the genesis flow later.

Reply:
Sure